tests/smoke/smoketests.sh

```diff
@@ -139,6 +139,7 @@ for BACKEND in ${BACKENDS[@]}; do
 				echo "Test DISABLED; GitHub runner does not have enough memory for this test"
 			else
 				echo ">>>      [x]           [ ]       Tests HPCG on a small matrix"
+				echo "Functional test executable: ${TEST_BIN_DIR}/hpcg_${BACKEND}"
```

Comment:
I don't believe this line should be here, does line 144 actually not already have this effect? (Two options for fixing: either remove this line, or remove line 144-- slight preference for the latter actually for me)